### ISSUE TITLE
cannot assign user to report only role, cannot remove user from user role

### ISSUE THREAD

**Issue by dallenk** [iomad/iomad#1693]
Attempting to assign a user to a report only role gives this error:
`Coding error detected, it must be fixed by a programmer: Invalid call to role_unassign(), please use role_unassign_all() when removing multiple role assignments`

Cannot remove a user from the "user" role from /iomad_company_admin/company_managers_form.php
As Administrator:  No error is shown, no action is taken.
As a company manager:  `Coding error detected, it must be fixed by a programmer: Invalid call to role_unassign(), please use role_unassign_all() when removing multiple role assignments`

**Comment by dallenk** [comment]
update:  These errors only appear to occur when the "Short name" or "Custom full name" is modified in the "Manage roles" section of the Site Administration /admin/roles/manage.php

When I changed these items to reflect a more accurate name to the roles the errors happen for Company Manager, and Company Department Manager.  Returning these values to the original short and long names resolves the error.

Something is missing in the code somewhere..  @turf212 are you monitoring these issues??   I've tried to report a couple security issues I have discovered but have not seen a reply from you yet.

Thanks
